### ISSUE TITLE
ID was missing the `signalR` element

### ISSUE THREAD

**Issue by dimm91** [pulumi/pulumi-azure#473]
I'm  Trying to create a Signal R resource using the azure provider, but even though it got created, Pulumi throws an error.

### Program.cs
```
static Task<int> Main()
    {
        return Deployment.RunAsync(() => {
            
            // Create an Azure Resource Group
            var resourceGroup = new ResourceGroup("resourceGroup");
            
            //Creates a Signal R service
            var SignalR= new Pulumi.Azure.SignalR.Service($"Signalrfoo-dev",new Pulumi.Azure.SignalR.ServiceArgs(){
                ResourceGroupName= resourceGroup.Name,
                Sku = new Pulumi.Azure.SignalR.Inputs.ServiceSkuArgs{
                    Capacity = 1,
                    Name = "Free_F1",
                }
            });
            
            return new Dictionary<string, object?>
            {
                {"SignalR:ConnectionString", SignalR.PrimaryConnectionString}
            };
        });
    }
```
### Pulumi Version:
- v1.11.1

### Expected Behaviour:
- Pulumi should not report any error and should return the connection string as an output as stated on the code

### Actual Behaviour:

- Pulumi creates the resource on azure, but returns this error: 
        * creating urn:pulumi:dev::Test-Pulumi-with-Git::azure:signalr/service:Service::signalrfoo-dev: ID was missing the `signalR` element

### Steps to reproduce:

1. Create new Pulumi Project with `azure-csharp` template
2. Paste the code or create a Signal R Service 
3. Run the command `pulumi up` and confirm or `pulumi up --yes`

**Comment by mikhailshilkov** [comment]
Hi @dimm91 

I ran your code and got

```
Do you want to perform this update? yes
Updating (dev):
     Type                           Name            Status      
     pulumi:pulumi:Stack            azure-cs-dev                
 +   ├─ azure:signalr:Service       Signalrfoo-dev  created     
 
Outputs:
  + SignalR:ConnectionString: "Endpoint=https://signalrfoo-dev673ffb64.service.signalr.net;AccessKey=<redacted>=;Version=1.0;"

Resources:
    + 1 created
    1 changes. 2 unchanged

Duration: 4m19s
```

Are you sure this is the exact code that you used?

**Comment by dimm91** [comment]
Hi @mikhailshilkov, I did and, the result was the same mentioned earlier, pretty weird that we have different results.

 I'm attaching an image of the result.

![Screenshot 2020-03-04 at 17 40 21](https://user-images.githubusercontent.com/8717994/75901733-4615b600-5e3f-11ea-8a04-d45c14bf1241.png)

Can be that we are using different versions?

**Comment by mikhailshilkov** [comment]
I'm on
```
<ItemGroup>
    <PackageReference Include="Pulumi.Azure" Version="1.12.0-preview" />
  </ItemGroup>
```
You?

**Comment by dimm91** [comment]
I'm on

```

  <ItemGroup>
    <PackageReference Include="Pulumi.Azure" Version="1.14.0-preview" />
  </ItemGroup>
 
```

**Comment by mikhailshilkov** [comment]
A-ha... reproduced with `1.14`! This appears to be this issue https://github.com/terraform-providers/terraform-provider-azurerm/issues/5715
It's going to be fixed in 2.0 of the provider (to be released soon).
